oresat_star_tracker/utils.py

```diff
@@ -0,0 +1,44 @@
+import unittest
```

Comment:
What is this for?

Reply:
This was just a bunch of helpers for the test but can be merged into the test.